src/editor/adapters/in-memory-editor.ts

```diff
@@ -59,8 +63,8 @@ export class InMemoryEditor implements Editor {
     return Promise.resolve();
   }
 
-  async writeIn(path: RelativePath, code: Code): Promise<void> {
-    this.otherFiles.set(path, new InMemoryEditor(code));
+  async writeIn(path: Path, code: Code): Promise<void> {
+    this.otherFiles.set(path as RelativePath, new InMemoryEditor(code));
```

Comment:
Is it necessary to force the cast here? I'll have a look.

Reply:
With this https://github.com/nicoespeon/abracadabra/pull/725#discussion_r1009991380 changes, cast is not required anymore. 

Changes:
- Remove cast and fix typos https://github.com/nicoespeon/abracadabra/pull/725/commits/5695d2da50fa8f4e98090552ce59361bd4da43b4
-  Use AbsolutePath in test too https://github.com/nicoespeon/abracadabra/pull/725/commits/d2d8425a4370673ec502bdd74c5ace5026ac23e9